data/data/bootstrap/baremetal/files/etc/containers/systemd/ironic-httpd.container.template

```diff
@@ -17,10 +17,15 @@ Network=host
 # podman (rather than systemd) redirect the logs to journald.
 PodmanArgs=--log-driver=journald
 Volume=ironic.volume:/shared:z
+{{ if not .PlatformData.BareMetal.DisableIronicVirtualMediaTLS }}
+Volume=/opt/openshift/tls/ironic/ironic-virtualmedia-tls.crt:/certs/vmedia/tls.crt:z
+Volume=/opt/openshift/tls/ironic/ironic-virtualmedia-tls.key:/certs/vmedia/tls.key:z
```

Comment:
nit: I think Zane was suggesting to rename the files into just tls.crt and tls.key, then you can mount the entire /opt/openshift/tls/ironic/ into /certs/vmedia/

Reply:
These file names no longer match what you're actually generating.

Reply:
Fixed